src/datastar_py/attributes.py

```diff
@@ -433,6 +433,19 @@ def throttle(
         return self
 
 
+class DelayMod:
+    def delay(
+        self: Self,
+        wait: int | float | str,
+    ) -> Self:
+        """Delay the event listener.
+
+        :param wait: The minimum interval between events.
```

Comment:
Description is a bit wrong here, this isn't an interval between events.

Reply:
Addressed